lib/ddtrace/contrib/action_view/configuration/settings.rb

```diff
@@ -0,0 +1,24 @@
+require 'ddtrace/contrib/configuration/settings'
+require 'ddtrace/contrib/action_view/ext'
+
+module Datadog
+  module Contrib
+    module ActionView
+      module Configuration
+        # Custom settings for the ActionView integration
+        class Settings < Contrib::Configuration::Settings
+          option  :analytics_enabled,
+                  default: -> { env_to_bool(Ext::ENV_ANALYTICS_ENABLED, false) },
+                  lazy: true
+
+          option  :analytics_sample_rate,
+                  default: -> { env_to_float(Ext::ENV_ANALYTICS_SAMPLE_RATE, 1.0) },
+                  lazy: true
+
+          option :service_name, default: Ext::SERVICE_NAME
```

Comment:
Are there cases when `action_view` will be the top level service?

I wonder if we need to even set a service name at all? Should this just be grouped under the Rails service? or should we have as something like:

```ruby
option :service_name, default: ENV['DATADOG_SERVICE_NAME'] ? "#{ENV['DATADOG_SERVICE_NAME']}-#{Ext::SERVICE_NAME}" : Ext::SERVICE_NAME
```

Reply:
We'll get this default or the Rails service name by default.